src/sql/workbench/contrib/notebook/browser/cellViews/code.css

```diff
@@ -80,11 +90,12 @@ code-component .carbon-taskbar .codicon {
 }
 
 code-component .carbon-taskbar .codicon.hideIcon {
+	color: #999999;
```

Comment:
What's the purpose of setting the color to this?

Reply:
This affects the symbol: [ ] sitting in the query editor toolbar column. Colors for this will also need to be themed.

Reply:
Fixed by adding registerThemingParticipant to code component and removed from CSS.